### ISSUE TITLE
Accessibility on seamless access default page

### ISSUE THREAD

**Issue by cami-dev** [TheIdentitySelector/thiss-js#307]
Hi! Our organization is using seamless access to select identity provider and we've found a few accessibility problems when looking through the page ourself. There might be more but would be great if you could have a look at these. I read on your documentation that you aspire to fullfill WCAG 2.1 so would be awesome if you could have a look at these problems below.

### **What we found:**

**High prio:**

* The text on the page can not resized https://www.w3.org/WAI/WCAG21/Understanding/resize-text.html
since the body text-size is set in pixels. Needs to be rem. So browsers can modify accordingly to the users settings.

* When searching for a provider the info about search results is not announced in the screen reader. Probably due to the aria-live tag being on the actual element which is shown. Try moving it up to the element above. It needs to always be in the DOM.

<img width="1444" height="813" alt="Image" src="https://github.com/user-attachments/assets/0ccc9915-0ce2-4ac5-bbad-1c79dab1726b" />

### **Still important and needed for WCAG compliant:**

* Language of page tag is not set depending on selected language. It always shows en. 
[https://github.com/TheIdentitySelector/thiss-js/blob/staging/src/ds/index.ejs](https://github.com/TheIdentitySelector/thiss-js/blob/staging/src/ds/index.ejs) row 2 
According to https://www.w3.org/WAI/WCAG22/Understanding/language-of-page.html this needs to be dynamic if other parts of the page is supposed to be in another language.

* Alt text on the page logo is not good it says: "RA 21 Logo "which is not descriptive or helpfull. In this case it might be best to just leave the logo alt text blank since it's neither a clickable link nor anything that brings purpose to a screen reader. Or just Seamless Access logo or something more human.

* The text before the page name is not read in the screen reader. It seems focus is set to the page name when loaded. It would be nice to include the text "Access to" aswell.

<img width="754" height="563" alt="Image" src="https://github.com/user-attachments/assets/ecbd66d9-06a3-4b34-9ee1-c543d4828460" />

* The labels for the search input field on "Find my organization is in english even though another language has been selected. 
https://github.com/TheIdentitySelector/thiss-js/blob/staging/src/ds/index.ejs row 62 they need to be dynamic aswell. Same goes for the aria-label.

* The aria-describedby is not correctly connected to the description below the field they need to have an id that is the same.

<img width="1238" height="811" alt="Image" src="https://github.com/user-attachments/assets/3e9178da-8212-4bfa-a03a-866f5aa0f3d0" />

* The text show when pressing "Learn more" resets focus on the page to the top so it's very hard knowing what the button does. Also the button to close that infobox can't be accessed through keyboard. 

<img width="587" height="586" alt="Image" src="https://github.com/user-attachments/assets/73f37aab-37b5-4ea9-9a63-45a5cf4c957a" />

* The remove button only says "Remove" which seem hardcoded in as a title. This would need to be language specific. It also does not contain which item I'm removing. Should be something like Remove - item name  

<img width="611" height="474" alt="Image" src="https://github.com/user-attachments/assets/a57af462-f107-4033-a870-33c493571608" />

* The heading order is wrong there is a H2 before a H1. Which makes screenreaders work poorly. And the order of the content when skipping between headings will be wrong.

<img width="1350" height="765" alt="Image" src="https://github.com/user-attachments/assets/f1a89f58-27b6-4ac0-92f2-e02c0e7cffa2" />

If you have any questions or want to discuss any of the above just give me a shout :)
Thanks for working with accessibility so we all can enjoy the web!

**Comment by sunetzacharias** [comment]
Thank you for this!

Noting one thing about the logo and alt text:
> Alt text on the page logo is not good it says: "RA 21 Logo "which is not descriptive or helpfull. In this case it might be best to just leave the logo alt text blank since it's neither a clickable link nor anything that brings purpose to a screen reader. Or just Seamless Access logo or something more human.

We are preparing to offer service providers the ability to use the top left corner to populate with their own logo.
In that case I'm not sure if it makes sense to have that logo name readable. I take your suggestion to leave it blank to be the best option for us.

**Comment by enriquepablo** [comment]
Thank You!
We will fix all these issues in an upcoming release, we still have no date for it but are already testing its new features.

**Comment by cami-dev** [comment]
Any estimate when fixes for these issue will be live? Is it a few weeks away or months? Need to give our product owner an answere for decision making in implementing the new login page.

**Comment by sunetzacharias** [comment]
Hello again! (_Some time in the next two weeks the changes will come into production._)
You can sign up the status newsletter and get information about when there is a release scheuled here:
https://status.seamlessaccess.org/

Edit: I have to confirm with the rest of the team which of the fixes are part of the new release, which will be fixed later, and if any which will not be fixed.

**Comment by cami-dev** [comment]
Awesome :) thx a lot! Will check it out.

**Comment by sunetzacharias** [comment]
All the suggested fixes in regards to accessibility was released with the latest version @cami-dev
